test/parallel/test-readline-interface.js

```diff
@@ -331,8 +331,8 @@ function isWarned(emitter) {
   // over the default crlfDelay but within the setting value
   {
     const fi = new FakeInput();
-    const delay = 200;
-    const crlfDelay = 500;
+    const delay = common.platformTimeout(125);
```

Comment:
Would this be more reliable *without* `common.platformTimeout()` here but keeping it in the line below?